brayovsky/pivotaly
# Enable start story from sideview

#### What does this PR do? / Fixes issue
Adds start story button in backlog stories that either:
- Creates a new branch to start a story
- Starts story in a different existing branch
- Starts story in the same branch
- Starts story if workspace is not a git repo

#### How should this be manually tested?
Click on start story from the backlog view
Different options should appear to pick from

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
<img width="389" alt="Screen Shot 2020-02-24 at 19 38 05" src="https://user-images.githubusercontent.com/19430730/75171808-46c29400-573d-11ea-99d5-fd3839bcbdbd.png">


#### Questions:
